zdm/iteration.py

```diff
@@ -16,10 +17,10 @@
 #### this verson currently works with 5 params.
 # soon to be upgraded with two values of local DM contribution
 
-def get_likelihood(pset,grid,survey,norm=True,psnr=False):
+def get_likelihood(pset,grid,survey,norm=True,psnr=True):
```

Comment:
can you add to the doc string here, i.e. define the arguments including `psnr`
thx

Reply:
Yep, will do.

Reply:
Agreed, this should be the default

Reply:
Okay.